tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -4373,18 +4373,21 @@ void IrEmitterUnnested::EmitPrologueForReduction(
 
 void IrEmitterUnnested::EmitFullWarpShuffleDownLoopForAllReduces(
     absl::Span<HloComputation* const> reducers,
-    absl::Span<llvm::AllocaInst* const> partial_result_addresses) {
+    absl::Span<llvm::AllocaInst* const> partial_result_addresses,
+    int block_size) {
```

Comment:
Any reason to not say `thread_per_block` to be consistent with the `GetThreadsPerBlock`?

Reply:
I amended the commit.